Quran.xcodeproj/project.pbxproj

```diff
@@ -1143,7 +1160,7 @@
 				TargetAttributes = {
 					54B265E21CC57D8E00D3AEF8 = {
 						CreatedOnToolsVersion = 7.3;
-						DevelopmentTeam = MKP73M5X4A;
```

Comment:
Can you revert all changes related to signing identities, dev team and bundle id?